src/main/java/seedu/address/model/person/Meeting.java

```diff
@@ -102,9 +97,19 @@ public boolean equals(Object o) {
 
     @Override
     public String toString() {
+        if (isEmpty()) {
```

Comment:
Should we be worried about an adversarial user deliberately inputting the max and min date?